Dockerfile

```diff
@@ -4,19 +4,21 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 ENV DEBCONF_NONINTERACTIVE_SEEN=true
 
-WORKDIR /opt/nautobot
+ENV NAUTOBOT_DATABASE="nautobot"
```

Comment:
Can we match the env vars used here: https://github.com/nautobot/nautobot/blob/develop/development/nautobot_config.py#L12-L18